schema/manifest.schema.json

```diff
@@ -104,16 +104,41 @@
       },
       "default": []
     },
+
     "proxy": {
-      "type": "object",
+      "type": ["object", "array"],
       "default": {},
-      "maxProperties": 4,
-      "patternProperties": {
-        ".*": {
-          "type": "string",
-          "format": "uri-reference"
+      "anyOf": [
+        {
+          "type": "object",
+          "maxProperties": 4,
+          "patternProperties": {
+            ".*": {
+              "type": "string",
+              "format": "uri-reference"
+            }
+          }
+        },
+        {
+          "type": "array",
+          "maxItems": 4,
```

Comment:
The same goes here. Why is the limit 4 proxies?

Reply:
See above